rasa/utils/io.py

```diff
@@ -257,23 +257,44 @@ def create_directory_for_file(file_path: Text) -> None:
             raise
 
 
-def questionary_file_path_validator(
+def file_type_validator(
     valid_file_types: List[Text], error_message: Text
 ) -> Type["Validator"]:
     """Creates a `Validator` class which can be used with `questionary` to validate
        file paths.
     """
 
+    def is_valid(path: Text) -> bool:
+        return path is not None and any(
+            [path.endswith(file_type) for file_type in valid_file_types]
+        )
+
+    return create_validator(is_valid, error_message)
+
+
+def not_empty_validator(error_message: Text) -> Type["Validator"]:
+    """Creates a `Validator` class which can be used with `questionary` to validate
+    that the user entered something other than whitespace.
+    """
+
+    def is_valid(input: Text) -> bool:
+        return input is not None and input.strip() != ""
+
+    return create_validator(is_valid, error_message)
+
+
+def create_validator(function: Callable, error_message: Text) -> Type["Validator"]:
```

Comment:
```suggestion
def create_validator(function: Callable[[Text], bool], error_message: Text) -> Type["Validator"]:
```
might even be `Optional[Text]]`

Reply:
i don't think it's optional because when i tried `None` as input. However, does that format mean that the input is only one `Text` input? because your validator has multiple inputs and mine only one